cgi/partial/ADDForm.tt

```diff
@@ -273,11 +273,31 @@ function popReviewInfo(user) {
 // Rewords the Commonwealth date entry based on whether "Pub Date" is checked.
 function toggleADD() {
   var label = document.getElementById("renewalFieldLabel");
-  var cb = document.getElementById("renewalFieldCheckbox");
+  var cb = document.getElementById("pub-checkbox");
   var field = document.getElementById("renewalField");
   label.innerHTML = (cb.checked) ? "Publication&nbsp;Date:" : "Author&nbsp;Death&nbsp;Date:";
   field.title = (cb.checked) ? "Publication Date" : "Author Death Date";
 }
+
+// Make sure pub date checkbox is checked if crown checkbox is checked
```

Comment:
These 2 functions could be just 1 function, if you pass the ids in as parameters
```
function syncFromXtoY(id_x, id_y) { ... }
```
and copy the checkedness from x to y.

Reply:
Had to have on extra parameter (Boolean copying checked vs unchecked) but yes this DRYed out the JS a bit.